packages/components/src/navigable-container/container.js

```diff
@@ -68,6 +68,11 @@ class NavigableContainer extends Component {
 		const { getFocusableContext } = this;
 		const { cycle = true, eventToOffset, onNavigate = noop, stopNavigationEvents } = this.props;
```

Comment:
```suggestion
		const { cycle = true, eventToOffset = noop, onNavigate = noop, stopNavigationEvents } = this.props;
```

This should be enough to make this work instead of the check:
> // This avoids a console error when eventToOffset is null